src/astt_gui/templates/index.html

```diff
@@ -201,22 +229,35 @@ <h1>CAM Dashboard</h1>
                 <div class="container">
                   <div class="row">
                     <div class="col">
-                     <h4>Stow-Pin State</h4>
-                     <button class="btn btn-secondary" style="width: 190px;" id="stowState">UNKNOWN</button>
-                    </div>
+                      <h4>Stow-Pin State</h4>
+                      <button class="btn btn-secondary" title="This is where we display our stow states" style="width: 190px;" id="stowState">UNKNOWN</button>
+                      <!--<div class="btn-container">
+                        <button class="btn btn-secondary" id="stowState">UNKNOWN</button>
```

Comment:
Oops , Dead code.
Needs some cleaning.

Reply:
This is what I was trying when I wanted to style the description but consulting Zach, he explained that it is not possible but I wanted surprise to also try from his side, if you can see I have new CSS file and a JS file check files changed.